src/eckit/exception/Exceptions.h

```diff
@@ -8,6 +8,20 @@
  * does it submit to any jurisdiction.
  */
 
+/**
+ * @file Exceptions.h
```

Comment:
I would remove "@file" as it can quickly become outdated, and it is unclear in respect to other "Exceptions.h" files (such as in codec/ and geo/)